hi_shalom.py

```diff
@@ -0,0 +1,11 @@
+"""
+This program reads a sentence and returns a list of all the words length.
+"""
+
+def words_length(string):
+    words = string.split()
+    words_len = [len(word) for word in words]
+    return words_len
+
+
+print(words_length("Toto, I've a feeling we're not in Kansas anymore"))
```

Comment:
File should end with a newline - 
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline